cli/src/pcluster/aws/aws_resources.py

```diff
@@ -183,6 +183,36 @@ def gpu_count(self):
 
         return gpu_count
 
+    def gpu_manufacturers(self):
+        """Return the list of GPU manufacturers supported by this instance type."""
+        gpu_info = self.instance_type_data.get("GpuInfo", None)
+
+        if gpu_info:
+            return list({gpu.get("Manufacturer") for gpu in gpu_info.get("Gpus", [])})
+        return []
+
+    def inference_accelerator_names(self):
```

Comment:
`inference_accelerator_manufacturer` without the `s`. I do not expect an instance type have more than 1 inference accelerator of different manufacturers.

Thus, this should return a single value, not a list.